hdk-rust/src/api.rs

```diff
@@ -341,14 +341,13 @@ pub enum BundleOnClose {
 //--------------------------------------------------------------------------------------------------
 
 /// Call an exposed function from another zome or another (bridged) instance running
-/// on the same agent in the same conductor.
-/// Arguments for the called function are passed as `JsonString`.
-/// Returns the value that's returned by the given function as a json str.
+/// in the same conductor.
+/// Arguments for the called function are passed and resturned as `JsonString`.
 /// # Examples
-/// In order to utilize `call`, you must have at least two separate Zomes.
-/// Here are two Zome examples, where one performs a `call` into the other.
+/// Here are two example Zomes, where one performs a `call` into the other.
```

Comment:
Thoughts on 'zome' -- I see mixed capitalisation through the documentation; should we settle on Zome or zome?

Reply:
Dunno what's best.  Probably Zome?

Reply:
Good question. Grammatically I'd lean towards 'zome' because we often use it as a common noun (e.g., 'a zome'). But it's an odd word that we want to call attention to (e.g., so people don't misread it as 'zone').